.github/workflows/publish_latest.yml

```diff
@@ -0,0 +1,90 @@
+name: Publish Latest Cycamore
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/publish_latest.yml'

```

Comment:
Something should run when only the `Dockerfile` changes

Reply:
See cymetric discussion, I believe `build_test.yml` should run when only the `Dockerfile` changes